### PR TITLE
Clean up README.md

### PR DESCRIPTION
**Pull request by adaszko**
README.md is the first point of contact with the project.  It is our landing page.  It is very important it remains clean and lucid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,87 +173,33 @@ rnode {
...
+We use YourKit to profile rchain performance.  YourKit supports open source
+projects with its full-featured Java Profiler.  YourKit, LLC is the creator of
+<a href="https://www.yourkit.com/java/profiler/">YourKit Java Profiler</a> and
+<a href="https://www.yourkit.com/.net/profiler/">YourKit .NET Profiler</a>,
 innovative and intelligent tools for profiling Java and .NET applications.
 
 ## Licence information
 
-To get summery of licenses being used by the RChain's dependencies, simply run `sbt node/dumpLicenseReport`. The report will be available under `node/target/license-reports/rnode-licenses.html`
+To get summery of licenses being used by the RChain's dependencies, simply run
```
Comment (odeac) [inline README.md]: typo "summary"

File: README.md [hunk 2]
```diff
@@ -114,26 +107,25 @@ The above command should result in (`rnode0` output):
...
-[... compiling, testing, jarring ...]
-[info] Packaging /home/kirkwood/src/rchain/rholang/target/scala-2.12/rholang-assembly-0.1.0-SNAPSHOT.jar ...
-<computer:~/src/rchain (dev)> rholang/rho2rbl rholang/examples/hello_world_again.rho
-compiled rholang/examples/hello_world_again.rho to rholang/examples/hello_world_again.rbl
-<computer:~/src/rchain (dev)> cat rholang/examples/hello_world_again.rbl; echo
-(let [[helloworld (generateFresh "helloworld")]] (block ( (proc [] (run (compile (let [[[[Rholanga4716fe347be]] (consume t [helloworld] [['world]] #t)]] ( (proc [[world]] (let [[[Rholang5401f0efc120] (consume t [world] ['msg] #f)]] ( (proc [[[msg]]] ( display msg "
-")) [Rholang5401f0efc120]))) [Rholanga4716fe347be])))))) (let [[world (generateFresh "world")] [world2 (generateFresh "world2")]] (block (block (block (produce t helloworld world) (produce t world "Hello World")) (produce t helloworld world2)) (produce t world2 "Hello World again")))))
-<computer:~/src/rchain (dev)>
-```
+To get a full list of options rnode accepts, use the `--help` option: `docker
```
Comment (odeac) [inline README.md]: we should have all the commands written in the same style like 
```bash 
$ some command
```

**Comment by adaszko** [comment]
> @adaszko , love the updates! Why the deletion of the section on monitoring starting at line 219?

That wasn't the right place for them.  README should say how to install and run the software primarily.   That removed section belongs to features.md.

**Pull request merged**